### PR TITLE
Check ntp remove unused variables

### PR DESCRIPTION
**Pull request by RincewindsHat**
compiler told me this variable was unused, so I removed it.
Also formatted the code and fixed a typo

### CODE REVIEW

File: plugins/check_ntp.c [hunk 1]
```diff
@@ -153,748 +165,784 @@ typedef struct {
...
+  printf("\tstatus: %d (0x%.2x)\n", ntohs(p->status), ntohs(p->status));
+  printf("\tassoc: %d (0x%.2x)\n", ntohs(p->assoc), ntohs(p->assoc));
+  printf("\toffset: %d (0x%.2x)\n", ntohs(p->offset), ntohs(p->offset));
+  printf("\tcount: %d (0x%.2x)\n", ntohs(p->count), ntohs(p->count));
+  numpeers = ntohs(p->count) / (sizeof(ntp_assoc_status_pair));
+  if (p->op & REM_RESP && p->op & OP_READSTAT) {
+    peer = (ntp_assoc_status_pair *)p->data;
+    for (i = 0; i < numpeers; i++) {
+      printf("\tpeer id %.2x status %.2x", ntohs(peer[i].assoc),
+             ntohs(peer[i].status));
```
Comment (sni) [inline plugins/check_ntp.c:293]: tbh, i don't think this is more readable than the existing code.

File: plugins/check_ntp.c [hunk 2]
```diff
@@ -153,748 +165,784 @@ typedef struct {
...
+  /* setup hints to only return results from getaddrinfo that we'd like */
+  memset(&hints, 0, sizeof(struct addrinfo));
+  hints.ai_family = address_family;
+  hints.ai_protocol = IPPROTO_UDP;
+  hints.ai_socktype = SOCK_DGRAM;
+
+  /* fill in ai with the list of hosts resolved by the host name */
+  ga_result = getaddrinfo(host, "123", &hints, &ai);
+  if (ga_result != 0) {
+    die(STATE_UNKNOWN, "error getting address for %s: %s\n", host,
```
Comment (sni) [inline plugins/check_ntp.c:409]: same here

**Comment by RincewindsHat** [comment]
Removed the reformatting, much smaller now.

**Review by sni — APPROVED** [review]
looks good, thanks

**Pull request merged**